squiggy/models/asset.py

```diff
@@ -46,22 +46,26 @@ class Asset(Base):
     asset_type = db.Column('type', assets_type, nullable=False)
     course_id = db.Column(db.Integer, nullable=False)
     description = db.Column(db.Text)
+    source = db.Column(db.String(255))
     title = db.Column(db.String(255))
     url = db.Column(db.String(255))
     visible = db.Column(db.Boolean, nullable=False)
+    categories = db.relationship(AssetCategory.__name__)
```

Comment:
Suggestion welcome on SQLAlchemy's best practice for mapping tables. This works but `categories` ends up as a list of `AssetCategory` objects.